chwifi

```diff
@@ -85,6 +85,34 @@ update_routine() {
     fi
 }
 
+update_profiles() {
+    printf '%s\n' "Updating profiles..."
+    config=$(config_dir)
+    profiles=""
+    # Grab other profiles and concatenate in string, using semicolon as seperator
+    for entry in "/etc/netctl"/*; do
+        if [ -f "$entry" ]; then
+            profiles+=$(echo "$entry" | sed -e 's/\/etc\/netctl\///g')';'
```

Comment:
I think using `,` instead of `;` would be nicer to conform to - sort of - a CSV syntax.

Reply:
e24bc52  fixes this.